### DISCUSSION TITLE
v3.0-beta3

### DISCUSSION THREAD

**Discussion by Tornaco** [Tornaco/Thanox#347] [General]
### What's Changed

1. Splash启动页适配Android12
2. 智能冻结：可以设置某个App是否检测到Activity启动请求自动解冻
3. 修复 #345 #346 在Android12上设备异常重启问题

<hr /><em>This discussion was created from the release <a href='https://github.com/Tornaco/Thanox/releases/tag/v3.0-beta3'>v3.0-beta3</a>.</em>

**Comment by Sfssup** [comment]
![Screenshot_2021-10-29-21-45-03-638_github.tornaco.android.thanos.jpg](https://user-images.githubusercontent.com/32392479/139446169-adf5ed3e-1d98-4dcf-975f-804388d7d1c4.jpg)
首页状态栏黑啦
安卓9，MIUI11
